src/Symfony/Component/Form/Extension/Validator/Type/UploadValidatorExtension.php

```diff
@@ -55,4 +56,19 @@ public function getExtendedType()
     {
         return 'form';
     }
+
+    public function __toString()
+    {
+        return $this->translator->trans($this->errorMessage, array(), $this->translationDomain);
+    }
+
+    /**
+     * This method is public because it needs to be callable from a closure in PHP 5.3.
```

Comment:
you can do without it by using a reference in the `use` of the closure, wouldn't that be better?